packages/victory-box-plot/src/victory-box-plot.test.tsx

```diff
@@ -159,7 +159,7 @@ describe("components/victory-box-plot", () => {
           q3Component={
             <Border
               ariaLabel={({ datum }) => buildLabel(datum.x, datum._q3)}
-              tabIndex={({ index }) => index + 1}
+              tabIndex={({ index }) => (index as number) + 1}
```

Comment:
I'm always hesitant to cast with `as`, since it's basically like saying "I _know_ that this is a number", without actually checking that it's a number.

In this case, `index` has a type of `string | number`, so I think it's more appropriate to do `+index + 1` to do a _runtime_ conversion to number instead of declaring that it's a number at the type level.